Worktez/worktez
# feat(contributors):'added static contributors page, also fixed some u…

…i bugs'
Signed-off-by: Abhishek Mishra [am1426620@gmail.com](mailto:am1426620@gmail.com)
### Functionality:
to add contributors page in static format

### Solution:
created the contributors page and fixed some UI bugs

## WtId: Dev649

### Risk level:

### How to test:
in the footer click contributors, in docs page and blogs pages check UI in dark mode
